kitodo-mediaserver-core/src/main/java/org/kitodo/mediaserver/core/conversion/SimpleIMSingleFileConverter.java

```diff
@@ -78,9 +90,9 @@ public InputStream convert(File master, Map<String, String> parameter) throws Ex
                 try {
                     watermarker.perform(operation, master, size);
                 } catch (Exception e) {
-                    String message = "Error creating watermark: " + e;
-                    // TODO notify?
+                    message = "Error creating watermark on file " + master + ": " + e;
```

Comment:
use master.getAbsolutePath() to print the whole path

Reply:
I added it, thanks.